### ISSUE TITLE
Unable to disable checks due to 'Unknown user script xxxxx'

### ISSUE THREAD

**Issue by 61swong** [ntop/ntopng#6986]
**Environment**:
* OS name: RHEL
* OS version: 7.9
* Architecture: x86_64
* ntopng version/revision: v5.4.221122

**What happened**:
When I tried to disable the remaining checks (left behind after 'Disable All'. I experienced error message 'Unknown user script "fortinet"'.

**How did you reproduce it?**
Restart ntopng and remove the enabled services.

**Debug Information**:
Screenshot of the error
![image](https://user-images.githubusercontent.com/50309519/203671971-bc7bd3ed-fc10-47ae-91c9-d17f312695e8.png)

**Comment by MatteoBiscosi** [comment]
Hi @61swong try out in this way:
- Click in Factory Reset (in the same page, next to disable all)
- Then Click Disable All again
and let me know if it disabled them or not

**Comment by 61swong** [comment]
Hi @MatteoBiscosi,

In dev release, after factory reset, I tried 'disabled all' and the relevant 5 checks were still there. I could disable them one by one. 

![image](https://user-images.githubusercontent.com/50309519/203872884-f48afb77-e057-4187-b838-f435ec2fbf32.png)

In Stable release, I did the factory reset and the 5 checks remained. However I couldn't disable them one by one with the same error message.

![image](https://user-images.githubusercontent.com/50309519/203872824-5d86dacd-5b11-4cfa-8fd8-30d0dc9a937b.png)
